lib/Ubic/UA.pm

```diff
@@ -57,7 +57,8 @@ sub get {
     my $socket = IO::Socket::INET->new(
         PeerAddr => $authority,
         Proto    => 'tcp',
-        Timeout  => $self->{timeout}
+        Timeout  => $self->{timeout},
+        GetAddrInfoFlags => 0
```

Comment:
Please explain more deeper why you disable all flags in getaddrinfo() and what flags are set by default.

Reply:
I disable only one AI_ADDRCONFIG flag, used by default

> If hints.ai_flags includes the AI_ADDRCONFIG flag, then IPv4
       addresses are returned in the list pointed to by res only if the
       local system has at least one IPv4 address configured, and IPv6
       addresses are returned only if the local system has at least one IPv6
       address configured.  The loopback address is not considered for this
       case as valid as a configured address.  This flag is useful on, for
       example, IPv4-only systems, to ensure that getaddrinfo() does not
       return IPv6 socket addresses that would always fail in connect(2) or
       bind(2).